engines/stark/resources/textureset.cpp

```diff
@@ -169,6 +171,15 @@ Gfx::TextureSet *TextureSet::readOverrideDdsArchive() {
 		// used by the models.
 		Common::String textureName = Common::String(name.c_str(), name.size() - 4);
 
+		// Fix for loading one of Emma's textures in mods caused by inconsistent zip filename encoding
+		//  The original game uses a 1 byte character (\xe6 which is ae in CP1252)
+		//	\x91 is ae in CP437 and CP850 which is used by some compression utilities
+		//	\xc3\xa6 is the UTF-8 2-byte representation
+		// Both get converted to the game's original encoding here
+		if (textureName == "pupp\x91""r" || textureName == "pupp\xc3\xa6""r") {
```

Comment:
Why are you doubling the quote here?
It doesn't seem to be needed as per below.
Either you separate the "r" in every places either you don't.

Reply:
> Why are you doubling the quote here? It doesn't seem to be needed as per below. Either you separate the "r" in every places either you don't.

I implemented feedback from @sev- , personally I think it would be more readable to have it in one block but will go with the project consensus. Would a single set of quotes be preferred?

Reply:
single quote is preferred

Reply:
Ok cool, that's done in the latest version of the branch.

Reply:
Fixed in new PR https://github.com/scummvm/scummvm/pull/5251 as Github closed this one after I rebased my branch.